content/developer/blueprints/spec-relationships.md

```diff
@@ -38,136 +43,119 @@ source_interfaces| no       | dict        | A dictionary of interfaces.
 target_interfaces| no       | dict        | A dictionary of interfaces.
 
 <br>
-
 By default, nodes can be related using the relationship types described below. You may also [declare your own](#declaring-relationship-types) relationship types.
 
-## The *cloudify.relationships.depends_on* Relationship Type
+## Fundamental Built-in Relationships
 
-Describes a node that depends on another node. For example, the creation of a new subnet depends on the creation of a new network.
+There are three fundamental relationships defined by Cloudify, which underlie all other relationship types, including those defined in plugins or blueprint authors.  The base/root relationship type is `cloudify.relationships.depends_on`, which indicates processing order to the built-in workflows.    All relationships, even custom relationships, must ultimately derive from the depends_on relationship. Derived from this relationship are `cloudify.relationships.connected_to`, and `cloudify.relationships.contained_in`.
 
-{{% note title="Note" %}}
-The `cloudify.relationships.depends_on` relationship type is for use as a logical representation of dependencies between nodes. You should only use it in very specific cases when the other two relationship types are not relevant, or when you want to specify that a certain node should be created before or after another for the sake of ordering.
-{{% /note %}}
+![Basic Relationships](/images/blueprint/basic_rels.png)
 
-The other two relationship types inherit from the `cloudify.relationships.depends_on` relationship type. The semantics of the `cloudify.relationships.connected_to` relationship type is the same. Therefore, usage reference should be dictated by `cloudify.relationships.connected_to` which is described below.
+cloudify.relationships.contained_in indicates that the source node is “inside” the target node.  Often this is used to indicate that a node representing a software package is to be placed on a node representing a compute instance (or instances).  Each node template can only have a single contained_in relationship.  This limitation is enforced universally by the DSL parser ( i.e. it isn’t a workflow specific quality ).  If the relationship target is derived from cloudify.nodes.Compute, the built in workflows will understand this to mean that the source node instance is to be remotely installed on target instance.
```

Comment:
It will also expect an agent to be installed by default, unless agent_config cancels that.